### DISCUSSION TITLE
Códigos warehouse

### DISCUSSION THREAD

**Discussion by PabloDevGit** [josantonius/php-mercadona-importer#3] [General]
Hola, no he podido probarlo aún, estoy valorando todo pero parece un gran trabajo el tuyo, una pregunta, estos códigos warehouse ¿cómo puedo obtener la lista completa de códigos? quiero decir, los códigos para todas las comarcas o provincias etc...

**Comment by josantonius** [comment]
Hola! 

En su momento no encontré una manera simple para obtener los códigos, no sé si seguirá igual a día de hoy.

Cuando accedes a un producto desde la web se envía una petición a la API que incluye el código del almacén:

`https://tienda.mercadona.es/api/products/3601/?lang=es&wh=mad1`

El almacén variará dependiendo del código postal de tu dirección de entrega. Por lo tanto, conociendo los [códigos postales](https://ayuda.tienda.mercadona.es/hc/es/articles/360001935371--A-qu%C3%A9-c%C3%B3digos-postales-llega-el-nuevo-servicio-de-compra-online-) donde reparten, podrías recorrerlos uno a uno e ir recopilando `provincia + código postal + almacén`.
